server/plugin/utils.go

```diff
@@ -187,7 +187,7 @@ func (p *Plugin) IsProjectLinked(projectList []serializers.ProjectDetails, proje
 
 func (p *Plugin) IsSubscriptionPresent(subscriptionList []*serializers.SubscriptionDetails, subscription *serializers.SubscriptionDetails) (*serializers.SubscriptionDetails, bool) {
 	for _, a := range subscriptionList {
-		if a.ProjectName == subscription.ProjectName && a.OrganizationName == subscription.OrganizationName && a.ChannelID == subscription.ChannelID && a.EventType == subscription.EventType && a.Repository == subscription.Repository && a.TargetBranch == subscription.TargetBranch && a.PullRequestCreatedBy == subscription.PullRequestCreatedBy && a.PullRequestReviewersContains == subscription.PullRequestReviewersContains && a.PushedBy == subscription.PushedBy && a.MergeResult == subscription.MergeResult && a.NotificationType == subscription.NotificationType && a.AreaPath == subscription.AreaPath && a.BuildPipeline == subscription.BuildPipeline && a.BuildStatus == subscription.BuildStatus && a.StageName == subscription.StageName && a.ReleasePipeline == subscription.ReleasePipeline && a.ReleaseStatus == subscription.ReleaseStatus && a.ApprovalType == subscription.ApprovalType && a.ApprovalStatus == subscription.ApprovalStatus {
+		if a.ProjectName == subscription.ProjectName && a.OrganizationName == subscription.OrganizationName && a.ChannelID == subscription.ChannelID && a.EventType == subscription.EventType && a.Repository == subscription.Repository && a.TargetBranch == subscription.TargetBranch && a.PullRequestCreatedBy == subscription.PullRequestCreatedBy && a.PullRequestReviewersContains == subscription.PullRequestReviewersContains && a.PushedBy == subscription.PushedBy && a.MergeResult == subscription.MergeResult && a.NotificationType == subscription.NotificationType && a.AreaPath == subscription.AreaPath && a.BuildPipeline == subscription.BuildPipeline && a.BuildStatus == subscription.BuildStatus && a.StageName == subscription.StageName && a.ReleasePipeline == subscription.ReleasePipeline && a.ReleaseStatus == subscription.ReleaseStatus && a.ApprovalType == subscription.ApprovalType && a.ApprovalStatus == subscription.ApprovalStatus && a.RunPipeline == subscription.RunPipeline && a.RunStageName == subscription.RunStageName && a.RunEnvironmentName == subscription.RunEnvironmentName && a.RunStageNameID == subscription.RunStageNameID && a.RunStageStateID == subscription.RunStageStateID && a.RunStageResultID == subscription.RunStageResultID && a.RunStateID == subscription.RunStateID && a.RunResultID == subscription.RunResultID {
```

Comment:
format it into multiline for easy readability, similar to its done in `handleGetSubscriptions`